src/main/kotlin/com/wgslfuzz/core/Resolver.kt

```diff
@@ -1593,14 +1592,17 @@ private fun resolveTypeOfFunctionCallExpression(
                     } else {
                         when (
                             val textureType =
-                                resolverState.resolvedEnvironment.typeOf(functionCallExpression.args[0]).asStoreTypeIfReference()
+                                resolverState.resolvedEnvironment
```

Comment:
Is there a functional change here? If not I'd leave the formatting as it was to make the diff cleaner.